ocm_python_wrapper/cluster.py

```diff
@@ -471,6 +471,39 @@ def __init__(self, client, cluster_name, addon_name):
     def addon_info(self):
         return self.client.api_clusters_mgmt_v1_addons_addon_id_get(self.addon_name).to_dict()
 
+    def get_addon_parameters_dict(self, _addon_parameters):
```

Comment:
any reason that `_addon_parameters` is private?

Reply:
no, changed